crmsh/utils.py

```diff
@@ -952,14 +952,14 @@ def is_int(s):
 
 
 def is_process(s):
-    cmd = "ps -ef -o comm= | grep -v grep | grep -qsE '\b%s\b'" % s
-    if options.regression_tests:
-        print ".EXT", cmd
-    proc = subprocess.Popen(cmd,
-                            shell=True,
-                            stdout=subprocess.PIPE)
-    proc.wait()
-    return proc.returncode == 0
+    commands = set()
+    pids = [pid for pid in os.listdir('/proc') if pid.isdigit()]
+    for pid in pids:
+        try:
+            commands.add(os.path.basename(open(os.path.join('/proc', pid, 'cmdline'), 'rb').read().replace('\x00', ' ').split(' ')[0]))
+        except os.error:
+            pass
+    return s in commands
```

Comment:
Yes, I think I prefer reading `/proc` since it is less error-prone than parsing the command output, but maybe I can make the code clearer :)